.github/workflows/test.yml

```diff
@@ -0,0 +1,122 @@
+---
+name: Test
+
+on:
+  pull_request:
+  push:
```

Comment:
```suggestion
```

Perhaps skip running tests on merges to master?

Reply:
Are there any disadvantages to running CI tests on master? The publish workflow runs independently.

Advantages of master CI tests are:
- Don't have to worry about PRs being out of sync with master. Since CI tests are run when the PR is updated if running the tests on master is an easy way to handle this possiblity.
- Easy to find record of all CI runs. When debugging a CI failure it is often useful to have a known good reference for comparison. If they're not run against master you'll have to go through the PR runs and possibly cross reference them with a commit. Similarly if an external change causes CI to start failing or a test to become flakier it's useful to hav the record of master CI runs to see when the change occured

Reply:
Deal let's retain it!